assets/collapsible-content.css

```diff
@@ -82,6 +86,7 @@
 .collapsible-row-layout .accordion {
   border: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
   margin-bottom: 1.5rem;
+  background-attachment: local;
```

Comment:
Added this to avoid this difference: 
![](https://screenshot.click/24-18-00yjy-exsue.png)